scripts/website.sh

```diff
@@ -4,5 +4,8 @@
 curl -g https://us-central1-clarity-design-system.cloudfunctions.net/actions -o .env
 # dotenv loads up the env variables into the shell, then deploys through Netlify
 
+# Move dev app into website
+mv -f ./dist/dev ./dist/website/dev
```

Comment:
Do we want this to be accessible after published? I think the netlify.toml needs a redirect for it

```
[[redirects]]
  from = "/dev/*"
  to = "/dev"
  status = 200
```